src/chains/ethereum/ethereum/src/forking/state-manager.ts

```diff
@@ -1,13 +1,13 @@
-import { Address as EJS_Address } from "ethereumjs-util";
-import { decode } from "rlp";
+import { Address as EJS_Address } from "@ethereumjs/util";
+import { decode } from "@ganache/rlp";
 import { ForkCache } from "./cache";
 import { ForkTrie } from "./trie";
 import { GanacheStateManager } from "../state-manager";
 
 /**
  * Options for constructing a [[GanacheStateManager]].
  */
-export interface GanacheStateManagerOpts {
+export interface ForkStateManagerOpts {
```

Comment:
I changed this name - it doesn't really have anything to do with the merge. I just thought that it was weird that we were calling it `GanacheStateManagerOpts` when it specifies a trie of type `ForkTrie`.